brave/brave-core
# Updates ads disabled content, bumps brave-ui

Fixes: https://github.com/brave/brave-browser/issues/2846

Removes unnecessary icons, brave-ui update includes changes to the DisabledContent component

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
1. Navigate to `brave://rewards` with Ads disabled
2. Confirm that "Coming Soon" is removed and replaced with new copy per design
3. Confirm latest megaphone icon per design is being used

## Reviewer Checklist:
